plugins/outputs/cloud_pubsub/README.md

```diff
@@ -0,0 +1,61 @@
+# Google Cloud PubSub Output Plugin
+
+The GCP PubSub plugin publishes metrics to a [Google Cloud PubSub][pubsub] topic
+as one of the supported [output data formats][].
+
+
+### Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage pubsub`.
+
+```toml
+[[inputs.pubsub]]
+  ## Required. Name of Google Cloud Platform (GCP) Project that owns
+  ## the given PubSub subscription.
+  project = "my-project"
+
+  ## Required. Name of PubSub subscription to ingest metrics from.
+  subscription = "my-subscription"
+
+  ## Required. Data format to consume.
+  ## Each data format has its own unique set of configuration options.
+  ## Read more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
+  data_format = "influx"
+
+  ## Optional. Filepath for GCP credentials JSON file to authorize calls to 
+  ## PubSub APIs. If not set explicitly, Telegraf will attempt to use 
+  ## Application Default Credentials, which is preferred. 
+  # credentials_file = "path/to/my/creds.json"
+
+  ## Optional. PubSub attributes to add to metrics.
+  # [[inputs.pubsub.attributes]]
+  #   my_attr = "tag_value"
```

Comment:
Move this option to the end, if it is uncommented the remaining options will fall under it in the TOML document.